source/oauthsso/requirements.rst

```diff
@@ -44,6 +44,10 @@ The fields that can be retrieved by SSO are as follows:
    :alt: field to complete
    :scale: 45 %
 
+.. Important::
+   If you are using **SCIM + OAuth SSO**, we recommend that you **leave these fields blank**. SCIM will take care of filling them in and will therefore not conflict with OAuth SSO, 
+   which would also try to fill in these fields. You could then end up with empty fields.
```

Comment:
```suggestion
   If you are using **SCIM + OAuth SSO**, we recommend that you **leave these fields blank**. SCIM will take care of filling them and will therefore not conflict with OAuth SSO.
```

Reply:
Not sure, but seems enough. Maybe wait for Curtis advices.

Reply:
With knowing only minimal information about both plugins, this seems sufficient:
If you are using **SCIM + OAuth SSO**, we recommend that you **leave these fields blank**. The fields will be automatically filled by the SCIM plugin later.

If I am wrong about how the plugins work, then the previous suggestion also works.

Reply:
I've had to change the sentence because it's now wrong. Romain has fixed a problem with the OAuth SSO plugin, so the behaviour is now more expected.